config/default/forms/contact/clinic-create.xml

```diff
@@ -1004,12 +1004,12 @@ DOB ISO: <output value=" /data/contact/ephemeral_dob/dob_iso "/></value></text>
       <bind nodeset="/data/inputs/user/facility_id" type="string"/>
       <bind nodeset="/data/inputs/user/name" type="string"/>
       <bind nodeset="/data/init/create_new_person" type="select1" required="true()"/>
-      <bind nodeset="/data/init/select_person" type="db:person" relevant="selected( /data/init/create_new_person ,'old_person')" required="true()"/>
+      <bind nodeset="/data/init/select_person" type="db:person" required="true()" relevant="selected( /data/init/create_new_person ,'old_person')"/>
       <bind nodeset="/data/init/name" type="string"/>
       <bind nodeset="/data/init/contact_name" type="string" calculate="coalesce(../../contact/name, ../name)"/>
-      <bind nodeset="/data/init/custom_place_name_label_translator" type="select1" relevant="selected( /data/init/create_new_person ,'none')" calculate="&quot;name&quot;"/>
+      <bind nodeset="/data/init/custom_place_name_label_translator" type="select1" calculate="&quot;name&quot;" relevant="selected( /data/init/create_new_person ,'none') or selected( /data/clinic/is_name_generated , 'false')"/>
       <bind nodeset="/data/init/custom_place_name_label" type="string" calculate="jr:choice-name( /data/init/custom_place_name_label_translator ,' /data/init/custom_place_name_label_translator ')"/>
-      <bind nodeset="/data/init/custom_place_name" type="string" relevant="selected( /data/init/create_new_person ,'none') or selected( /data/clinic/is_name_generated , 'false') or selected( /data/clinic/is_name_generated , 'no')" required="true()"/>
+      <bind nodeset="/data/init/custom_place_name" type="string" required="true()" relevant="selected( /data/init/create_new_person ,'none') or selected( /data/clinic/is_name_generated , 'false')"/>
```

Comment:
These two lines are the change. (Same chance for each file)  Basically, `custom_place_name_label_translator` was only relevant if `Skip this step` was selected for the initial "Set the Primary Contact" question. But the `custom_place_name_label_translator` gets used as the label for `custom_place_name` even if the user selects `Create a new person` or `Select an existing person` **IF** they also select `No, I want to name it manually` on the next page.  So, I updated `custom_place_name_label_translator` to be relevant in either case (and now its `relevant` value properly matches `custom_place_name`). 

I removed `or selected( /data/clinic/is_name_generated , 'no')` from the existing `relevant` value for `custom_place_name` since `no` is not a valid value for that question.